js/src/forum/addVoteButtons.js

```diff
@@ -47,6 +52,7 @@ export default function () {
                     },
                     loading: this.voteLoading,
                     disabled: this.voteLoading || !canVote,
+                    title: !canVote ? app.translator.trans('fof-gamification.forum.no_autovote_message') : '',
```

Comment:
Doesn't this assume the reason why the user can't vote is because they can't vote their own post? 

Also, I think we would want to change the key name to make it clearer.

Reply:
Yeah, you're right. User may not vote a post for other reasons.

How should we change this?

Reply:
Not 100% sure. The message should only be shown if they would normally be able to vote but can't vote on their own posts - you'd need to have extra variables to store the vote permission, whether they can vote for that specific post because of self voting, and the resulting conclusion of whether they can vote on the post or not.

I think check should probably be done in a custom policy on the PHP side though... and just not show a reason because if they can vote for other posts then they'll probably figure it out.

Reply:
How can I do the check in PHP? I haven't found anything related to permissions in the `src` directory...

Reply:
@datitisev can we look into this? Maybe we can display the message only when selfvoting is disabled

Reply:
I'm going to resolve this since I had no answer from @datitisev